src/components/directory-content.js

```diff
@@ -774,29 +787,44 @@ const DirectoryContent = () => {
         let children = getSelectedChildren();
         return (
             children.length === 1 &&
-            children[0].type === elementType.FILTERS_CONTINGENCY_LIST
+            children[0].type === elementType.CONTINGENCY_LIST &&
+            childrenMetadata[children[0].elementUuid].subtype ===
```

Comment:
maybe you should check if `childrenMetadata[children[0].elementUuid]` exists here first.
This is dirty code... OMG...

Reply:
Done.
Yes it's complicated... Maybe we should get the available menu actions from the back.

Reply:
It's not your dirty code :sweat_smile: